src/util/ActiveShapeUtils.tsx

```diff
@@ -14,7 +15,7 @@ import { GraphicalItem } from '../chart/generateCategoricalChart';
  *  - a render prop(inline function that returns jsx)
  *  - a react element
  */
-type ShapeType = 'trapezoid' | 'rectangle' | 'sector';
+type ShapeType = 'trapezoid' | 'rectangle' | 'sector' | 'symbols';
```

Comment:
Is it possible to elaborate in the JSDoc above what each of these values mean?

I'm reading _"It will handle taking in: *  - an object of svg properties *  - a boolean *  - a render prop(inline function that returns jsx) *  - a react element"_ but then the type itself is a union of strings? What's the relationship?

Reply:
I see how this comment is misleading. My intention was to explain purpose of the entire section of code from lines 18 - 85 in the initial commit. 

I've updated the comment to be clearer. Feel free to leave further feedback. 
```javascript
/**
 * This is an abstraction for rendering a user defined prop for a customized shape in several forms.
 *
 * <Shape /> is the root and will handle taking in:
 *  - an object of svg properties
 *  - a boolean
 *  - a render prop(inline function that returns jsx)
 *  - a react element
 *
 * <ShapeSelector /> is a subcomponent of <Shape /> and used to match a component
 * to the value of props.shapeType that is passed to the root.
 *
 */
```